package.json

```diff
@@ -36,6 +36,7 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "@rails/webpacker": "^5.2.1",
+    "webpack-merge": "^5.7.3",
```

Comment:
Merge conflict is caused by reordering of `package.json`, relevant also because this is out of order. It might fix itself automatically with a `yarn add webpack-merge`, or you could manually move it down.

Reply:
Whoops, fixed!